Exercise_B/index.html

```diff
@@ -30,5 +29,64 @@
+    </div>
+    <div class="upper">
+      <h1 id="headline">Beautiful Flowers</h1>
+      <div class="col-2 d-grid gap-2 mx-auto">
+        <button id="learnmore" class="btn btn-success" type="button">Learn more</button>
+      </div>
+      <!-- <p id="learnmore">Learn more</p> -->
+    </div>
+    <!--Lower-->
+    <!-- <div class="lower">
+      <div class="container">
+        <div class="row text-center d-flex p-1 p-1">
+          <div class="col-12 col-xl-3 p-1 d-inline-flex justify-content-start align-items-center">
+            <img class="img-fluid text-left" src="./assets/images/flower-icon.png" id="flower icon" alt="flower-icon" style="object-fit:contain;">
+            <p class="text-wrap" id="flower">Flowers for every kind of situation</p>
+          </div>
+          <div class="col-12 col-xl-3 p-1 d-flex justify-content-start align-items-center">
+            <img class="img-fluid text-left" src="./assets/images/payment-icon.png" id="payment icon" alt="payment-icon" style="object-fit:contain;">
+            <p class="text-wrap" id="payment">All major payments kinds accepted</p>
+          </div>
+          <div class="col-12 col-xl-3 p-1 d-flex justify-content-start align-items-center">
+            <img class="img-fluid text-left" src="assets/images/phone-icon.png" id="phone icon" alt="contact method" style="object-fit:contain;">
+            <p class="text-wrap" id="phone">Free consulation</p>
+          </div>
+          <div class="col-12 col-xl-3 p-1 d-flex justify-content-start align-items-center">
+            <img class="img-fluid text-left" src="assets/images/truck-icon.png" id="delivery icon" alt="delivery method" style="object-fit:contain;">
+            <p class="text-wrap" id="delivery">City wide delivery</p>
+          </div>
+        </div>
+      </div>
+    </div> -->
+
+    <div class="lower">
+      <div class="container text-center p-3">
+        <div class="row">
+          <div class="col-12 col-md-6 col-xl-3 p-1 d-inline-flex align-items-center">
+            <img class="" src="./assets/images/flower-icon.png"  alt="flower-icon">
+            <p class="mx-auto" id="flower">Flowers for very kind of situation</p>
+          </div>
+          <div class="col-12 col-md-6 col-xl-3 p-1 d-inline-flex align-items-center">
+            <img class="" src="./assets/images/payment-icon.png"  alt="payment-icon">
+            <p class="mx-auto" id="payment">All major payments kinds accepted</p>
+          </div>
+          <div class="col-12 col-md-6 col-xl-3 p-1 d-inline-flex align-items-center">
+            <img class="" src="assets/images/phone-icon.png" alt="contact method">
+            <p class="mx-auto" id="phone">Free consulation</p>
+          </div>
+          <div class="col-12 col-md-6 col-xl-3 p-1 d-inline-flex align-items-center">
+            <img class="" src="assets/images/truck-icon.png"  alt="delivery method">
+            <p class="mx-auto" id="delivery">City wide delivery</p>

```

Comment:
Great use of the bootstrap column system here it is well implemented and works well!

If you comment out code, either delete it or put a label ontop using comments denoting why its still in your code base

Reply:
Yes, should do...just sometimes I think I will re-do it later..so I didn't delete them and may use them as reference later...